### DISCUSSION TITLE
Merge Words for Dummies

### DISCUSSION THREAD

**Discussion by mayimagpie** [scribeocr/scribeocr#66] [General]
Is it possible to merge words split between 2 lines?

**Comment by Balearica** [comment]
No, it is not possible to merge words that are on separate lines.

What is the motivation behind this request? Are lines incorrectly being split in the middle of a word for certain documents?

**Reply by mayimagpie** [reply]
Sorry for the late reply! I was hoping to merge words that are split between lines (with hyphens) so the terms are still searchable.
